extensions/ql-vscode/src/remote-queries/remote-queries-markdown-generation.ts

```diff
@@ -330,3 +335,10 @@ function createFileName(nwo: string) {
   const [owner, repo] = nwo.split('/');
   return `${owner}-${repo}`;
 }
+
+/**
+ * Escape characters that could be interpreted as HTML instead of raw code.
+ */
+function escapeHtmlCharacters(text: string): string {
+  return text.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;');
```

Comment:
tl;dr what's here is fine 👍 

Out of curiosity I wondered about the performance of doing multiple regex `replace` calls. Turns out it is pretty good. Possibly the characteristics will change for a longer string, but I think this test case is roughly representative of real data and also I'm confident it'll be fine anyway.

I tried an online benchmarking site to test against using `replaceAll` to avoid the regex argument, or a single `replace` that matches all cases and uses a function to determine the replacement. The results are as follows:
<img width="1501" alt="Screenshot 2022-08-16 at 14 29 49" src="https://user-images.githubusercontent.com/3749000/184891900-1a2a672e-b17a-4ca0-8d6b-217fd5749ddb.png">